app/models/concerns/conclusion_final_reviews/defaults.rb

```diff
@@ -7,6 +7,26 @@ module ConclusionFinalReviews::Defaults
     after_initialize :set_defaults, if: :new_record?
   end
 
+  def assign_duplicate_images_from_draft ids_images_duplicates
+    draft = ConclusionDraftReview.where(review_id: review_id).first
+
+    if draft
+      ids_images_duplicates.reverse_each do |id_image_duplicate|
+        aux_image = ImageModel.find id_image_duplicate
+        new_image = ImageModel.new
+        new_image.image = File.open aux_image.image.file.file
+
+        aux_annex = aux_image.imageable
+        annexes.each do |annex_duplicate|
+          if aux_annex.title == annex_duplicate.title && aux_annex.description == annex_duplicate.description
+            annex_duplicate.image_models << new_image
+            new_image.imageable = nil
+          end
+        end
+      end
```

Comment:
No se si funciona, pero yo decía hacer algo por el estilo:
```suggestion
      draft.annexes.each do |annex|
        new_annex = annexes.build annex.attributes.dup.merge('id' => nil, 'image_model_id' => nil)
        
        annex.image_models.each do |image_model|
          new_annex.image_models.build image_model.attributes.dup.merge('id' => nil)
        end
      end
```

Está sin probar, lo largué así medio de memoria... pero si no me fallan los cálculos eso te debería popular el annexes del final con una copia "semi liviana" de los del draft, los dos van a quedar apuntando a los mismos archivos, pero eso no debería ser un problema (ya lo hacemos así con los papeles de trabajo).